lib/middleware/util.ts

```diff
@@ -54,3 +54,16 @@ export function after(req: Interface.IOurRequest,
 {
     req.log.info({res: res}, 'Response sent.');
 }
+
+export function rejectHTTP(req: Interface.IOurRequest,
+                           res: Interface.IOurResponse,
+                           next: restify.Next): void
+{
+    if (!conf.get('https.enable')) {
```

Comment:
I agree that we should trust the underlying framework to tell us if it is a secure connection or not if possible.